spec.md

```diff
@@ -758,6 +762,16 @@ For success responses, the following fields are defined:
   "description": "Creating service (10% complete)."
 }
 ```
+or
+```
+{
+  "state": "succeeded",
+  "description": "Service created.",
+  "resource": {
+    "dashboard_url": "http://example-dashboard.example.com/9189kdfsk0vfnku"
+  }
```

Comment:
So will resource always be just dashboard_url? or will it be any opaque object? or will it be a set of define attributes that we will add to as we need new fields?